server/services/cohorts.ts

```diff
@@ -94,7 +94,7 @@ export const filterCohortParticipantsForUser = (cohortParticipants, user) => {
     // Remove participants hired outside of HA's region
     // participant.siteJoin is joined based on hired status' siteId
     return cohortParticipants.filter((participant) =>
-      user.regions.includes(participant.siteJoin.body.healthAuthority)
+      user.regions.includes(participant.siteJoin?.body?.healthAuthority)
```

Comment:
Some of this optional chaining could be avoided by adding typing to `cohortParticipants`, but this also works.